app/connectors/IncomeTaxViewChangeConnector.scala

```diff
@@ -46,6 +46,10 @@ trait IncomeTaxViewChangeConnector extends RawResponseReads {
   val config: FrontendAppConfig
   implicit val ec: ExecutionContext
 
+  def getBusinessDetailsUrl(nino: String): String = {
+    s"${config.itvcProtectedService}/income-tax-view-change/income-sources/$nino"
```

Comment:
This won't work, the call to the backend will assume this is an mtd id
A seperate backend route will be needed

Reply:
fixed